staff/lab/mod-printer

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+"""Remove or add printers to/from CUPS classes."""
+import argparse
+import subprocess
+import sys
+
+
+def modify_printer(args):
+    """Perform internal call to CUPS CLI to modify class(es)."""
+    # Determine which CUPS classes are available
+    cups_classes = subprocess.check_output(["lpstat", "-p"]).decode("utf-8").split("\n")
+    # Remove empty trailing newline
+    cups_classes.remove("")
+    for i in range(len(cups_classes)):
+        clazz = cups_classes[i]
+        # Output is of the form: printer <printername> is idle. <etc>
+        clazz = clazz[len("printer "):]
+        clazz = clazz[:clazz.index(" is ")]
+        cups_classes[i] = clazz
+
+    if args.verbose:
+        print("Found CUPS classes: " + str(cups_classes))
+    
+    args_clazz = getattr(args, "class")
+    if args_clazz and args_clazz in cups_classes:
+        cups_classes = [args_clazz]
+        if args.verbose:
+            print("Overriding CUPS classes with parameter: " + str(args_clazz))
+    
+    # Perform pre-call logging and input processing
+    printer = args.printer.lower()
+    if args.verbose:
+        print("Printer to perform action on: " + printer)
+    action_flag = "-r" if args.action == "remove" else "-c"
+    if args.verbose:
+        print("Action to be performed (and flag): " + args.action, "(" + action_flag + ")")
```

Comment:
Use [f-strings](https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals), here and throughout.

Reply:
I normally would use f-strings, but in this case things seem to be running Python 3.5, which does not have support for f-strings. :(

Reply:
Does this only work on whiteout and not supernova?

If so then... ugh, yeah, forgot about that. Though I will say, whiteout is probably a relatively easy server to do a buster/bullseye upgrade on.

Reply:
Yeah... Maybe in the future I can do a buster/bullseye upgrade on whiteout, but at the moment that might be a bit much.